src/common/__tests__/kube-helpers.test.ts

```diff
@@ -72,55 +72,63 @@ describe("kube helpers", () => {
     });
     describe("with default validation options", () => {
       describe("with valid kubeconfig", () => {
-        it("does not raise exceptions", () => {
-          expect(() => { validateKubeConfig(kc, "valid");}).not.toThrow();
+        it("does not return an error", () => {
+          expect(validateKubeConfig(kc, "valid")).toBeUndefined();
         });
       });
       describe("with invalid context object", () => {
-        it("it raises exception", () => {
-          expect(() => { validateKubeConfig(kc, "invalid");}).toThrow("No valid context object provided in kubeconfig for context 'invalid'");
+        it("it returns an error", () => {
```

Comment:
While you're at it you could remove the extraneous "it" from the following test descriptions
```suggestion
        it("returns an error", () => {
```